release-notes.md

```diff
@@ -1,5 +1,11 @@
 # Service Broker API Release Notes #
 
+## <a id='2-12'></a>v2.12 ##
+2017-06-13
+
+* Added `context` field to request body for provisioning a service instance (`PUT /v2/service_instances/:instance_id`) and updating a service instance (`PATCH /v2/service_instances/:instance_id`). Also added a `profile.md` file describing recommended usage paterns for environment-specific extensions and variations.
+* The specification now uses RFC2119 keywords to make it clearer when things are required rather than optional or recommended.
```

Comment:
Are these intended to be complete?  I think everything in this list needs to be represented here: https://github.com/openservicebrokerapi/servicebroker/milestone/1?closed=1

https://github.com/openservicebrokerapi/servicebroker/pull/178 is an especially important one